etc/example-registry-auth.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- type: dockerhub
+  user: DOCKERHUB_USER
+  pass: DOCKERHUB_PASS
```

Comment:
This looks like it supports multiples by being an array but how do I know which registry this is for?